linker_scripts/code_script.txt

```diff
@@ -339,7 +339,7 @@ SECTIONS
         build/asm/z_camera.o(.text)
         build/src/code/code_0x800E03A0.o(.text)
         . += 0x10;
-        build/asm/z_collision_check.o(.text)
+        build/src/code/z_collision_check.o(.text)
```

Comment:
It looks like the build error is occurring because you added this file. Did you forget to commit `z_collision_check.c`?